src/lib/homestore_backend/hs_homeobject.hpp

```diff
@@ -490,7 +493,7 @@ class HSHomeObject : public HomeObjectImpl {
         pg_id_t pg_id_;
         shared< homestore::ReplDev > repl_dev_;
         uint64_t max_batch_size_;
-        std::unique_ptr<DonerSnapshotMetrics> metrics_;
+        std::unique_ptr< DonerSnapshotMetrics > metrics_;
```

Comment:
nit: Looks like the previous originator of these code doesn't have clang-format installed? Is it possible to kindly inform that we all have clang-format applied and make sure they are formatted on save?

Reply:
will do